packages/react-components/react-overflow/src/useOverflowContainer.ts

```diff
@@ -78,6 +79,20 @@ export const useOverflowContainer = <TElement extends HTMLElement>(
     [overflowManager],
   );
 
+  const registerDivider = React.useCallback(
+    (divider: OverflowDivider) => {
+      const el = divider.element;
+      overflowManager?.addDivider(divider);
```

Comment:
nit: do an early return the oveflowManager doesn't exist

Reply:
added to this issue #28236